xapian-core/tests/harness/testsuite.cc

```diff
@@ -713,16 +716,14 @@ test_driver::do_run_tests(vector<string>::const_iterator b,
 	    switch (test_res) {
 		case PASS:
 		    ++res.succeeded;
-		    if (verbose || !use_cr) {
-			out << col_green << " ok" << col_reset;
 
-			if (test_duration.count() >= SLOW_TEST_THRESHOLD) {
-			    out << col_yellow << " SLOW TEST ("
-				<< test_duration.count()
-				<< " ms)" << col_reset;
-			}
+		    if (test_duration.count() >= SLOW_TEST_THRESHOLD) {
+			slow_tests.push_back(make_pair(test->name,
+						       test_duration.count()));
```

Comment:
You can use `emplace_back()` and avoid the `make_pair()`.

Reply:
replaced push_back with emplace_back.